src/core/Launcher.jsx

```diff
@@ -1,16 +1,21 @@
 import React, { StrictMode, useState, useMemo } from 'react';
 import PropTypes from 'prop-types';
 
-import { SnackbarContext } from './contexts';
+import { SnackbarContext, WorldWeatherContext } from './contexts';
 
 function Launcher({ children }) {
   const [snackbar, setSnackbar] = useState({ isOpen: false, message: '' });
   const memoizedSnackbar = useMemo(() => [snackbar, setSnackbar], [snackbar]);
```

Comment:
why do you use useMemo here?

Reply:
Because the value of the state is an object. So that there would be no problems with re-rendering when changing the value to the same and the like. You can create 2 states with primitive values, but it's easier